string command
---
geometry_msgs/Point32 point
